cmd/loop/loopout.go

```diff
@@ -36,6 +36,13 @@ var loopOutCommand = cli.Command{
 			Name:  "amt",
 			Usage: "the amount in satoshis to loop out",
 		},
+		cli.Uint64Flag{
+			Name: "conf-target",
```

Comment:
Why no underscore?

Reply:
Hyphens are usually the standard for flags containing multiple words, but we seem to have different patterns (no separator, underscore, hyphen) across all of our CLI flags within `lnd`, `lncli` and `loop`.

Reply:
Most of the `lncli` flags use undercores. Only for `lnd` flags do we sometimes use a hyphen.